prisma/schema.prisma

```diff
@@ -143,6 +143,7 @@ model TrainingQueueMembership {
   joinedAt DateTime @default(now())
 
   @@id([userId, queueId])
+  @@unique([userId, queueId])
```

Comment:
a primary key is implictly already a unique index! (didn't realize it was a composite primary key, this is redundant 😄)